### DISCUSSION TITLE
How to create dynamic query of graphQL from schema

### DISCUSSION THREAD

**Discussion by renu944** [graphql-java/graphql-java#2292] [Q&A]
Hi ,
I have a requirement to generate dynamic query of GraphQL from schema. I need to create dynamic schema and its query by passing table name from the database. I am able to create schema dynamically. Can anyone help me to create dynamic queries on it? 

Thanks

**Comment by TcPaulyue** [comment]
I have the same problem with you.in my way,i parse the schema by myself,then add schemaDefinition info in typeRegistry and runtimewiring,finally rebuild the graphQLSchema from updated typeRegistry and runtimewiring. you can see my implement in https://github.com/TcPaulyue/jsonSchema-to-GraphQL
